src/scripts/create-parameters.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
You don't need a `#!` line, as the job will insert its own: https://app.circleci.com/pipelines/github/CircleCI-Public/path-filtering-orb/263/workflows/10998727-25cb-4fce-b85b-75ee603ef75a/jobs/1330?invite=true#step-102-1

Reply:
looks like it's needed for shellcheck job https://app.circleci.com/pipelines/github/CircleCI-Public/path-filtering-orb/264/workflows/0945eeef-3b65-415a-956f-fbfc9146ec59/jobs/1331